governance/program/src/state/enums.rs

```diff
@@ -155,12 +155,17 @@ pub enum VoteThreshold {
     /// The minimum number of votes in % out of the entire pool of governance tokens eligible to vote
     /// which must be cast for the vote to be valid
     /// Once the quorum is achieved a simple majority (50%+1) of Yes votes is required for the vote to succeed
-    /// Note: Quorum is not implemented in the current version
+    // TODO: if quorum is achieved how it's considered about veto and deny votes?
     QuorumPercentage(u8),
 
     /// Disabled vote threshold indicates the given voting population (community or council) is not allowed to vote
     /// on proposals for the given Governance
     Disabled,
+
+    /// The minimum number of votes in % out of the entire pool of governance tokens eligible to vote
+    /// which must be cast for the proposal being considered as successful
+    /// Once the quorum is achieved all options are considered as successful
+    QuorumPercentageAllSucceed(u8),
```

Comment:
This should not be defined here because it would make it possible to very easily create malicious proposals, deceive community and trick it into approving some harmful instructions.  
 
In general a proposal with this type of voting semantics  cannot be used to execute instructions per option. It can only be used as a poll to ask for opinions or preferences across the given options.

Because of the above this should be defined as a vote type for a proposal and not as governance rules threshold and the governance rules quorum would be then used to determine whether the poll is valid or not (ex. minimum required attendance).
For such a proposal we can allocate a single, proposal level executable instruction (as opposed to an instruction per option)  and then make ti possible to execute it if the poll is valid.